plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -173,7 +173,7 @@ class Bazel(
             "mod", "graph", "--output", "json", "--disk_cache=", workingDir = projectDir
         )
 
-        val node = JSON.decodeFromString<ModuleGraphNode>(modGraphProcess.stdout)
+        val node = JSON.decodeFromString<BazelModule>(modGraphProcess.stdout)
```

Comment:
@haikoschol, to my this sounds a bit too much like a [`MODULE.bazel` file](https://bazel.build/rules/lib/globals/module), which is something different, right?

Reply:
I struggle to have a strong opinion on this. :) Some random thoughts:

A MODULE.bazel file contains more (all?) information about a module whereas the nodes in this graph only contain name and version. Calling the node `BazelModule` somewhat overpromises on how much information it contains.

Currently the content of a MODULE.bazel file is represented by the class `ModuleMetadata`, which is not a great name either.

I included `GraphNode` in the name because the class seemed to be tightly coupled to this particular way of retrieving the dependency tree.

Ultimately I think `BazelModule` works as a name for either of those. Both classes are used only in a few lines in the code and not passed around to "faraway places" where it's unclear what they represent. If ORT ever reads MODULE.bazel files of dependencies and does a lot of processing on them, we might want to reconsider the naming.

Reply:
Ok, thanks for sharing your thoughts, let's stick to `BazelModule` then.